index.js

```diff
@@ -48,15 +49,26 @@ CloudSpeechRecognizer.startStreaming = (audioStream, cloudSpeechRecognizer) => {
 const Sonus = {}
 Sonus.init = (options, recognizer) => {
   // don't mutate options
-  const opts = Object.assign({}, options)
-
-  const sonus = new stream.Writable()
+  const opts = Object.assign({}, options),
+    models = new Models(),
+    sonus = new stream.Writable(),
+    csr = CloudSpeechRecognizer.init(recognizer)
   sonus.mic = {}
-  const csr = CloudSpeechRecognizer.init(recognizer)
+  
+  opts.hotwords = opts.hotwords || [1]
```

Comment:
It's a 1 line hack so that if someone passes in a blank config object we'll fallback to adding a single default model. When we iterate over the hotwords (in this case a single item array) it will create one for us.
It's the same as:
``` javascript
if(!opts.hotwords){
  models.add({
    file: 'node_modules/snowboy/resources/snowboy.umdl',
    sensitivity: '0.5',
    hotwords : 'default'
  })
} else {
  opts.hotwords.forEach(model => {
    models.add({
      file: model.file || 'node_modules/snowboy/resources/snowboy.umdl',
      sensitivity: model.sensitivity || '0.5',
      hotwords : model.hotword || 'default'
    })
  })
}
```